### ISSUE TITLE
A11y_code.visualstudio.com_VisualStudioCode_Linux_ColorContrast: Color Contrast ratio is less that 3:1 when focus goes to the snap store control.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode-docs#3726]
[Check out Accessibility Insights!](https://github.com/microsoft/AzureStorageExplorer/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS1.4.11;#Win10;#Visual Studio Code;#ColorContrast; #CAITestPass_May2020;#FTP;#STP;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.488.1 (Official build) dev (64-bit)
Chrome Version: Version 83.0.4103.61 (Official Build) (64-bit)
Screenreader version
Narrator, NVDA[2020.1]
**Repro Steps:**
1. Launch the application Url :https://code.visualstudio.com
2. Navigate to "Docs" link.
3. From left navigation, select "Setup".
4. Navigate to "Linux" link.
5.Navigate to Snap store link and check whether the color contrast ratio is less than 3:1 when focus present on the link.
**Actual:**
While navigating through keyboard,the color contrast ratio is less than 3:1 when focus present on the link.
This issue is reproduced with Chromium Edge+ Narrator and Chrome + NVDA.

**Expected:**
The color contrast ratio should be more than 3:1.

**Attachment link for Reference:**

![3726](https://user-images.githubusercontent.com/53940787/83135538-941b1c80-a103-11ea-9d46-4dedf454929a.png)

**Comment by gregvanl** [comment]
That image comes from the Snap Store so we don't have control over the contrast.

Also seems like sufficient contrast when tabbing to the link image

![image](https://user-images.githubusercontent.com/12818376/83205373-3c7cb100-a103-11ea-9137-343266b918ba.png)
.

**Comment by nirmalsahoo** [comment]
@gregvanl the color 2 should not be white. It should be the color on which snap store written. The border should be clearly visible.

**Comment by ManishaC1** [comment]
Github tags:#Closed
![Screenshot (444)](https://user-images.githubusercontent.com/105198219/217432680-25a6ffbe-af0e-46c6-85ff-571c27b6e6a4.png)

**Comment by ManishaC1** [comment]
Github tags:#Closed;

**Comment by sohelmulani1998** [comment]
#P1_A11y_code.visualstudio.com_Apr2020;
